resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/initcontainer/InitContainerConfigurationStepsOrchestrator.scala

```diff
@@ -62,6 +62,15 @@ private[spark] class InitContainerConfigurationStepsOrchestrator(
     submissionSparkConf.get(RESOURCE_STAGING_SERVER_INTERNAL_SSL_ENABLED)
       .orElse(submissionSparkConf.get(RESOURCE_STAGING_SERVER_SSL_ENABLED))
       .getOrElse(false)
+
+  OptionRequirements.requireSecondIfFirstIsDefined(
+    KubernetesFileUtils.getOnlySubmitterLocalFiles(sparkJars).nonEmpty match {
```

Comment:
Don't match on true or false here - use the functional APIs like `map`, `filter`, `getOrElse`, etc.